config/repo-values.yaml

```diff
@@ -78,3 +78,6 @@ sync:
       url : https://releases.rancher.com/server-charts/stable
     - name: loki
       url: https://grafana.github.io/loki/charts
+    - name: mmlspark
+      url: https://dbanda.github.io/charts
```

Comment:
I have some questions about this repo:

1. You've noted this has to do with mmlspark. But, the images referenced within the chart don't point to the official mmlspark images from Microsoft. Is there a reason for that?
2. The chart lists maintainers who, I don't believe, are mmlspart maintainers.
3. The maintainers listed in the other file are for mmlspark. Do you have a pointer that you're on that team so we can close the loop?

Reply:
Thanks for taking a look! Dalitso and I are both part of the listed alias as well as several other members of our team so that will be the best place to shoot queries about the chart.

Reply:
Is this necessary given https://github.com/helm/charts/pull/11285. It appears to be a fork of the same chart - why not update the upstream chart?

Reply:
@lachie83 thanks for the feedback. My undestanding is that the community could have access to the mmlspark repo through helm hub. Here we would host our spark chart and possibly other mmlspark charts. It would also allow us to make quick changes to the chart without the need to push to the stable upstream repository each time.

Reply:
Yeah it certainly could but the current chart that I see there has me listed as a maintainer and looks like the current upstream chart with the proposed changes in the other PR.

Reply:
@dbanda Your reasoning to move the chart to a separate repo makes sense - this is what we're encouraging chart owners to do in the long term (though since this is official, should it be github.com/Azure rather than github.com/dbanda)?

I also agree with @lachie83 that ideally we wouldn't have two versions of the same chart. In that spirit, what we usually do is deprecate the chart in helm/charts after adding to hub from an external, official source. When you do, could you start off by retaining the git history from the community helm/charts directory for this chart? (see my comment below https://github.com/helm/hub/pull/65#issuecomment-487132673)

Reply:
@scottrigby thanks for the feedback. We wanted to make this helm hub so that we could use this to release snapshot bits to customers on each minor version bump. We want to use the stable chat for the release bits so that it is discoverable to the spark community who is already using it.